src/app/shared/issue-tables/IssuesDataTable.ts

```diff
@@ -110,6 +110,8 @@ export class IssuesDataTable extends DataSource<Issue> {
           return this.compareValue(a.assignees.join(', '), b.assignees.join(', '));
         case 'teamAssigned':
           return this.compareValue(a.teamAssigned.id, b.teamAssigned.id);
+        case 'Todo Remaining':
+          return this.compareValue(this.todoFinished(a) - a.issueDisputes.length, this.todoFinished(b) - b.issueDisputes.length);
```

Comment:
What you essentially need is remaining number of todos right? You can have a method in issueDispute model to do this instead of calculating it here.